homeassistant/components/zwave_js/config_flow.py

```diff
@@ -352,7 +352,7 @@ async def async_step_usb(self, discovery_info: dict[str, str]) -> FlowResult:
         manufacturer = discovery_info["manufacturer"]
         description = discovery_info["description"]
         # Zooz uses this vid/pid, but so do 2652 sticks
-        if vid == "10C4" and pid == "EA60" and "2652" in description:
+        if vid == "10C4" and pid == "EA60" and description and "2652" in description:
```

Comment:
`description` can be None, according to the `USBDevice` dataclass, so I added a check on `description` before checking if it contains "2652".